packages/dev/core/src/Rendering/boundingBoxRenderer.ts

```diff
@@ -162,6 +178,29 @@ export class BoundingBoxRenderer implements ISceneComponent {
     private _uniformBufferFront: UniformBuffer;
     private _uniformBufferBack: UniformBuffer;
     private _renderPassIdForOcclusionQuery: number;
+    /**
+     * Internal buffer for instanced rendering
+     * @internal

```

Comment:
You don't need this tag, as the variable is already internal (private). Same thing for the `_useInstances` protected variable.

Reply:
Done in https://github.com/BabylonJS/Babylon.js/pull/15911/commits/d74bba6a40cbef506eeef1d8b21af5ab8b327441